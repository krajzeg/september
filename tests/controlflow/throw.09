nastyFunction := {
	throw: ENoMoreElements()
}

print("The function should throw an ENoMoreElements exception.")
try {
	nastyFunction()
	print("But it doesn't.")
} catch (ENoMoreElements) {
	print("And it did.")
}
