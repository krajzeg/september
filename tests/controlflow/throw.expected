The function should throw an ENoMoreElements exception.
And it did.
